handlers/david.js

```diff
@@ -0,0 +1,116 @@
+const _      = require('lodash');
+const format = require('format');
+const helpers = require('../util/helpers.js');
+
+module.exports = {
+  name: 'david',
+  trigger: '.david',
+  handler: david,
+};
+
+// refers to the odds of modifying an individual character
+// NOT the odds of that particular modification happening to a word
+const config = {
+  addCharChance: .3,
+  capitalizationSwapChance: .4,
+  scrambleChance: .4,
+  removeCharChance: .2,
+}
+
+async function david(message) {
+  const messages = await helpers.getMessages(message);
+  const randomMessage = helpers.randomUserMessage(messages).content;
+  let words = randomMessage.split(" ");
+
+  words = _.map(words, (word) => {
+    const fn = getRandomModifier();
+    return fn(word);
+  });
+
+  words.push(Math.random() > .5 ? "lol" : "haha");
+  const davidifiedString = words.join(" ");
+  message.channel.send("David: " + davidifiedString);
+}
+
+function getRandomModifier() {
```

Comment:
take a look at the sample function in lodash

https://lodash.com/docs/4.17.5#sample

should work for this since they all have equal weight